yt_idv/shaders/ray_tracing.frag.glsl

```diff
@@ -53,6 +158,22 @@ void spherical_coord_shortcircuit()
     vec3 dir = -normalize(camera_pos.xyz - ray_position_xyz);
     vec4 curr_color = vec4(0.0);
 
+    // intersections
+    vec2 t_sphere_outer = get_ray_sphere_intersection(right_edge[id_r], ray_position_xyz, dir);
+    if (t_sphere_outer[0] == INFINITY && t_sphere_outer[1] == INFINITY)
```

Comment:
this does make me a little itty bit nervous.  One of the weird semantics of `NaN` values is that equality checks are not reliable, by design.  I know you've constructed this to be `inf` (which does support equality, at least in numpy), but I am still sort of anxious.

```
In [3]: np.nan == np.nan
Out[3]: False
```

Reply:
ya! fair point! There is a glsl `isinf` that I should have used here, which would be better than the equality comparison. But generally, this gets at how I'm not super happy with how this is presently returning the t values... it's just using INFINITY as a flag here, so maybe a refactor would remove the comparison question entirely.

Reply:
now using isinf for this check